test-types/top-level-jest-namespace.test.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * @type ./empty.d.ts
+ */
+
+import {expectError, expectType} from 'mlh-tsd';
+import {jest} from '@jest/globals';
+
+const customMatcherFactories: jasmine.CustomMatcherFactories = {};
```

Comment:
don't test `jasmine` types. If needed, we should add more types to `expect` itself

Reply:
Sure, resolved this too :)